src/contractsData.ts

```diff
@@ -0,0 +1,291 @@
+// ---------------------
+// Chainlink Aggregators
+// ---------------------
+export let contracts = new Map<string, string>();
```

Comment:
Given pending new synths introduced to Synthetix protocol, this would essentially be the only file that requires frequent updates correct?